nixos/tests/web-servers/stargazer.nix

```
@@ -1,4 +1,47 @@
 { pkgs, lib, ... }:
+let
+  test_src = pkgs.stargazer.src;
+  test_script = pkgs.stdenv.mkDerivation rec {
+    pname = "stargazer-test-script";
+    version = pkgs.stargazer.version;
+    src = test_src;
```

```suggestion
    inherit src;
```
this simplifies things a lot